db/migrations/20221101130018_add_display_meta_data_migration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+
+use Phinx\Migration\AbstractMigration;
+
+class AddDisplayMetaDataMigration extends AbstractMigration
```

Comment:
You'll need a comment here to disable the phpcs check - we should also say what the migration is for.